### ISSUE TITLE
[CI] GC overhead limit exceeded

### ISSUE THREAD

**Issue by DaveCTurner** [elastic/elasticsearch#27898]
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+intake/885/console failed saying

```
:distribution:bwc-snapshot-6.1:buildBwcVersion FAILED
:distribution:bwc-snapshot-6.1:buildBwcVersion (Thread[Task worker for ':' Thread 3,5,main]) completed. Took 5 mins 32.4 secs.
1863 actionable tasks: 1814 executed, 49 up-to-date

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':checkout-6.1:core:compileJava'.
> GC overhead limit exceeded
```

This build apparently used Gradle 4.4, in case that's relevant.

**Comment by jasontedor** [comment]
I think this is a duplicate of #27892, at least it appears the JDK 9 in that issue is irrelevant now?

**Comment by DaveCTurner** [comment]
This is the first mainstream one I've come across to fail in this way, but yes, it now looks like it's not related to JDK9. I wasn't expecting to see any builds using Gradle 4.4 today so haven't been looking at the Gradle version until now, but I'll go back and see if that correlates.

**Comment by jasontedor** [comment]
Gradle 4.4 is everywhere in CI now.

**Comment by jasontedor** [comment]
Closed by 8b2eee39f06b5a13c998a1096c7791b404dc8473
